src/gen_dragonfruit/gd_dev_adafruit_MPL115A2_temp.h

```diff
@@ -1,7 +1,7 @@
 #include "Adafruit_MPL115A2.h"
 
-#ifndef _PIN_GD_ADAFRUIT_MPL115A2_TEMP_H
-#define _PIN_GD_ADAFRUIT_MPL115A2_TEMP_H
+#ifndef _GD_ADAFRUIT_MPL115A2_TEMP_H
```

Comment:
I would say at some point, we might consider change this device driver to reference adafruit to freescale.

The sensor is actually manufactured by freescale semiconductor (https://cdn-shop.adafruit.com/datasheets/MPL115A2.pdf). Adafruit just makes the breakout board that was referenced by the previous design team - we actually don't use a breakout board at all on the dragonfruit board.

It might be something that we have to talk about as a team sometime later - the cranberry drivers do not actually reference the manufacturer in the file/function names at all. I won't block the pull request for this at the moment, however.